.gitignore

```
@@ -92,3 +92,14 @@ sheets.json
 *.key
 *.cer
 
+# File generated by windows during development
+**/gp12-pem*
+**/is-ci*
+**/mime*
+**/mongo_dump*
+**/nodemon*
+**/nodetouch*
+**/nopt*
+**/rc*
+**/semver*
+**/uuid*
```

lets add another rule to ignore any files with the `.cmd` and `.ps1` extension
```suggestion
**/uuid*
*.cmd
*.ps1
```